lib/ansible/modules/system/filesystem.py

```diff
@@ -74,8 +75,28 @@
 import re
 import stat
 
-from ansible.module_utils.basic import AnsibleModule
-from ansible.module_utils.six import viewkeys
+from ansible.module_utils.basic import AnsibleModule, get_platform
+
+
+class Device(object):
+    def __init__(self, path):
+        self.path = path
+
+
+    def size(self):
```

Comment:
In order to avoid to execute command twice, size could be stored using something like:
```
def __init__(self, path):
    self._size = None
[...]
def size(self):
    if self._size is None:
        # compute self._size
        [...]
    return self._size
```

Reply:
Yes, but the cost is not much - maybe 1ms. I think the simple solution is better for now.